doc/source/cluster/kubernetes/k8s-ecosystem/scheduler-plugins.md

```diff
@@ -0,0 +1,55 @@
+# KubeRay integration with scheduler plugins
+
+The [kubernetes-sigs/scheduler-plugins](https://github.com/kubernetes-sigs/scheduler-plugins) repository provides out-of-tree scheduler plugins based on the scheduler framework.
+
+Starting with KubeRay v1.4.0, KubeRay integrates with the [PodGroup API](https://github.com/kubernetes-sigs/scheduler-plugins/blob/93126eabdf526010bf697d5963d849eab7e8e898/site/content/en/docs/plugins/coscheduling.md) provided by scheduler plugins to support gang scheduling for RayCluster custom resources.
+
+## Step 1: Create a Kubernetes cluster with Kind
+
+```sh
+kind create cluster --image=kindest/node:v1.26.0
+```
+
+## Step 2: Install Scheduler Plugins
+
+Follow [install.md](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/doc/install.md) in the scheduler-plugins repository to install the scheduler plugins.
+
+:::{note}
+
+Note that there are two modes for installing the scheduler plugins: **single scheduler mode** or **second scheduler mode**.
+
+For KubeRay v1.4.0, KubeRay only supports the **single scheduler mode**.
```

Comment:
Have you already tried both modes and confirmed that it only works with the single scheduler mode?

Reply:
I haven't tried the second scheduler mode. I am referring to https://scheduler-plugins.sigs.k8s.io/docs/user-guide/installation/:

<img width="718" alt="image" src="https://github.com/user-attachments/assets/41b8418c-5ee5-4311-847a-d9806558c63a" />

I can give it a try.

Reply:
It doesn't work with the second scheduler although Step 4 is the same.

I tried to deploy a RayCluster with 1 CPU for the head Pod and 1000 CPUs for the worker Pods. The head Pod will still be scheduled when gang scheduling is enabled.